acloud/converters.go

```diff
@@ -0,0 +1,6 @@
+package acloud
+
+// String convert
+func String(v string) *string {
```

Comment:
Is this functionality not already included in acloudapi or can you use generic pointers interface?

Like `pointers.String(..)` or even a generic solution like `pointers.Pointer(..)`

Reply:
I don't see it. There is only a Bool pointer method that is available

Reply:
Then i might be good to add a generic solution here.

```
// ToPtr returns a pointer to the given value.
func ToPtr[T any](v T) *T {
    return &v
}
```